process/sync/baseSync.go

```diff
@@ -740,7 +740,14 @@ func (boot *baseBootstrap) rollBack(revertUsingForkNonce bool) error {
 			return err
 		}
 
-		boot.outportHandler.RevertIndexedBlock(currHeader, currBody)
+		err = boot.outportHandler.RevertIndexedBlock(currHeader, currBody)
+		if err != nil {
+			log.Debug("boot.outportHandler.RevertIndexedBlock",
```

Comment:
here we return the error and log it as well, is there a reason?

Reply:
copied from the rest of the function. Removing the log.Debug